### PR TITLE
external/protobuf: Fix build script in a protobuf sample.

### PR DESCRIPTION
**Pull request by pillip8282**
protocol buffer sample program doesn't need grpc plugin build.
So all grpc plugin features in protobuf sample could be deleted.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/grpc_greeter_client/Kconfig [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 config EXAMPLES_GREETER_CLIENT
 	bool "grpc client example"
+    depends on GRPC
```
Comment (sunghan-chang) [inline apps/examples/grpc_greeter_client/Kconfig]: Could you align new line? Tab instead of spaces makes it align

Comment (pillip8282) [reply apps/examples/grpc_greeter_client/Kconfig]: Thanks for your comment. I will fix

Comment (pillip8282) [reply apps/examples/grpc_greeter_client/Kconfig]: And it will be fixed by another pull request so I will reset it

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**